api/app/views/mno_enterprise/config/show.js.coffee

```diff
@@ -3,13 +3,14 @@
 angular.module('mnoEnterprise.configuration', [])
   .constant('IMPAC_CONFIG', <%= Settings.impac.to_json.html_safe %>)
   .constant('I18N_CONFIG', {
-    enabled: <%= MnoEnterprise.i18n_enabled %>,
-    available_locales: <%= I18n.available_locales.to_json.html_safe %>
+    enabled: <%= Settings.system.i18n.enabled %>,
+    available_locales: <%= @available_locales.to_json.html_safe %>,
+    preferred_locale: <%= Settings.system.i18n.preferred_locale.to_json.html_safe %>
```

Comment:
maybe introduce `@preferred_local' for consistency ?

Reply:
Actually if you look at this template, `@available_locales`  is the exception. 
Everything else is coming directly from `Settings`. We have to do some formatting on the`@available_locales` hence the ivar.